code_samples/node_modules/ssl_sample.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var Config = require('hazelcast-client').Config;
+var HazelcastClient = require('hazelcast-client').Client;
+
+var clientConfig = new Config.ClientConfig();
+var sslConfig = new Config.SSLConfig();
```

Comment:
You can access the `sslConfig` over the `clientConfig`. There is no need to create another variable